test/evergreen/gcp_auth.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
I don't think we can have comments in json files. Can we add a FIXME to evergreen.yml to change `client_email` and `client_x509_cert_url` once we have the proper certs.

Reply:
I can't add a comment in json, I have made a ticket for this: [WT-10582](https://jira.mongodb.org/browse/WT-10582)

Reply:
Thanks! CC @jiechenbo in case you want to follow [WT-10582](https://jira.mongodb.org/browse/WT-10582)